lib/orm.php

```diff
@@ -2196,6 +2196,59 @@ public function save() {
 		return $success;
 	}
 
+	/**
+	 * Inserts multiple rows in a single query. Expects new rows as it's a strictly insert function, not an update one.
+	 *
+	 * @param array $models Array of model instances to be inserted.
+	 *
+	 * @return bool True for successful insert, false for failed.
+	 *
+	 * @example From the Indexable_Link_Builder class: $this->seo_links_repository->query()->save_many( $links );
+	 *
+	 * @throws \Exception Instance to be inserted is not a new one.
+	 */
+	public function insert_many( $models ) {
+		$values  = [];
+		$success = true;
+
+		foreach ( $models as $model ) {
+			if ( ! $model->orm->is_new ) {
+				throw new \Exception( 'Instance to be inserted is not a new one' );
+			}
+
+			// Remove any expression fields as they are already baked into the query.
+			$model_values = \array_values( \array_diff_key( $model->orm->dirty_fields, $model->orm->expr_fields ) );
+			$values       = \array_merge( $values, $model_values );
+		}
+
+		/**
+		 * Filter: 'wpseo_chunk_bulked_insert_queries' - Allow filtering the chunk size of each bulked INSERT query.
+		 *
+		 * @api int The chunk size of the bulked INSERT queries.
+		 */
+		$chunk = \apply_filters( 'wpseo_chunk_bulked_insert_queries', 1000 );
```

Comment:
Would make this `100` by default.

Reply:
Resolved in https://github.com/Yoast/wordpress-seo/pull/17364/commits/a97aa46285443491878fa2e93b457a77d01dcf94